otwcode/otwarchive
# AO3-5792 Add missing class to additional tag groups in Tag Sets to fix skins

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5792

The issue only mentions Reversi but Happy 17th! and Low Vision Default also have the wrong border color there.

## Purpose

Adds a missing "group" class to additional tag groups in Tag Sets so that they're targeted by Reversi, Happy 17th! and Low Vision Default site skins correctly.

This does not affect the default skin or most other public skins. Out of the three that are affected: Fixie (911) seems to get fixed by this as well, The Hustings (889) also becomes consistent (can't say "fixed" because the contrast there is terrible), and Panda Madness (890) doesn't account for these tag groups and has them look a bit out of place either way.

## Testing Instructions

1. Create a tag set with **31+** Additional tags _(the number in the Jira issue is incorrect)_
2. Switch skin to Reversi, or use ?site_skin=929 switch
3. The tag group headers in the Additional tags section should have dark backgrounds

## References

n/a

## Credit

slavalamp (they/them)
